app/containers/AccountInitializer/tests/saga.test.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Test sagas
+ */
+
+/* eslint-disable redux-saga/yield-effects */
+import { reviewAccountWorker } from '../saga';
+
+import { REVIEW_ACCOUNT_SUCCESS } from '../constants';
+
+describe('reviewAccountWorker Saga', () => {
+  const generator = reviewAccountWorker({});
+  it('Checking, step2 is action with REVIEW_ACCOUNT_SUCCESS type', () => {
```

Comment:
Need to check error and also all of the values coming back from saga.

https://github.com/react-boilerplate/react-boilerplate/blob/master/app/containers/HomePage/tests/saga.test.js